templates/digital-asset-template/frontend/pages/CreateCollection.tsx

```diff
@@ -1,20 +1,12 @@
 import { Button, buttonVariants } from "@/components/ui/button";
-import {
-  Card,
-  CardContent,
-  CardDescription,
-  CardHeader,
-} from "@/components/ui/card";
+import { Card, CardContent, CardDescription, CardHeader } from "@/components/ui/card";
```

Comment:
i run prettier but i still get these linter change from my auto formatter in vscode, weird

Reply:
also weird that it passes ci